### PR TITLE
Update ext compatibility for CiviCRM 5.57.1

### PR DESCRIPTION
**Pull request by andyburnsco**

### CODE REVIEW

File: info.xml [hunk 1]
```diff
@@ -14,11 +14,11 @@
     <url desc="Support">https://github.com/twomice/com.joineryhq.estatweaks/issues</url>
     <url desc="Licensing">http://www.gnu.org/licenses/gpl-3.0.html</url>
   </urls>
-  <releaseDate>2021-10-04</releaseDate>
-  <version>1.2-5.41.2</version>
+  <releaseDate>2021-1-12</releaseDate>
```
Comment (twomice) [inline info.xml]: Looks like a typo in the date format; also year should be 2023

**Comment by twomice** [comment]
Thanks @andyburnsco .

* I'm waiting to hear from esta on whether we should target 5.57.1 or 5.57.0 ; will update you on that.
* Once that's decided Could you also create a patch file at CRM_PATCHES/5.57.1.patch (or 5.57.0.patch, as appropriate). It should show the diff between our file overrides and the civicrm core files in that version. (I usually use `diff -uwp` for this, though I think there's a way to do it in git as well.)

**Comment by andyburnsco** [comment]
Patch file added though I fixed typos to reflect new version and added duplicate commits. Sorry, not sure on correcting existing commits...

**Comment by twomice** [comment]
Merged manually with `git merge --squash` to create a single commit. Thanks @andyburnsco !

**Pull request closed without merging**